build.gradle

```diff
@@ -220,53 +221,65 @@ artifacts {
 }
 
 task copyToLibs(type: Copy) {
-	into "$buildDir/libs"
-	from configurations.runtime
-    }
+    into "$buildDir/libs"
+    from configurations.runtime
+}
 
-task copyToRoot(type: Copy, dependsOn: [copyToLibs, jar, converterJar]) {
+task copyToRoot(type: Copy, dependsOn: [copyToLibs, jar, converterJar,
+        'unixLineEndings']) {
     outputs.files.setFrom(file("$buildDir/libs/pcgen-${version}.jar"))
 
-	from "$buildDir/libs/pcgen-${version}.jar"
-	from "$buildDir/libs/pcgen-${version}-batch-convert.jar"
-	from "$buildDir/release/pcgen.exe"
-	from "$projectDir/code/pcgen.sh"
-	into "$projectDir"
- 	rename "(.+)-$version(.+)", '$1$2'
+    from "$buildDir/libs/pcgen-${version}.jar"
+    from "$buildDir/libs/pcgen-${version}-batch-convert.jar"
+    from "$buildDir/release/pcgen.exe"
+    from "$projectDir/code/pcgen.sh"
+    into "$projectDir"
+    rename "(.+)-$version(.+)", '$1$2'
+}
+
+//Install the tools from: http://www.thefreecountry.com/tofrodos/
+task unixLineEndings( type: Exec) {
+    if(Os.isFamily(Os.FAMILY_WINDOWS)){
+        workingDir "$projectDir/code/sbin"
+        executable "bash"
+        args "-c", "fromdos $projectDir/code/pcgen.sh"
```

Comment:
Windows does not come with "bash" so I'm unsure of how this would execute on a windows machine...